src/templates/base/2019/base.html

```diff
@@ -99,11 +99,152 @@
 {% block contributors %}{{ config.contributors.keys() | length }}{% endblock %}
 
 {% block non_chapter_nav_links %}
-<li><a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.table_of_contents_title() }}</a></li>
 <li><a href="{{ url_for('contributors', year=year, lang=lang) }}">{{ self.contributors_title() }}</a></li>
 <li><a href="{{ url_for('methodology', year=year, lang=lang) }}">{{ self.methodology_title() }}</a></li>
 {% endblock %}
 
+{% macro nav_table_of_contents(switcher_name) %}
+  {% if switcher_name == "mobile" or switcher_name == "mobile-footer" %}
+    {{ native_table_of_contents_dropdown(switcher_name) }}
+  {% else %}
+    {{ styled_table_of_contents_dropdown(switcher_name) }}
+  {% endif %}
+{% endmacro %}
+
+{% macro native_table_of_contents_dropdown(switcher_name) %}
+{% set current_title = metadata.get('title') if metadata else None %}
+<div class="table-of-contents-switcher">
+  <label for="table-of-contents-switcher-{{switcher_name}}" class="visually-hidden">
+    {{ self.table_of_contents_title() }}
+  </label>
+  <select id="table-of-contents-switcher-{{switcher_name}}">
+    <option value="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.table_of_contents_title() }}</option>
```

Comment:
Actually I'm going to do one more change here as can't select ToC on mobile on home page (as that's the current page):

```suggestion
    <option value="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.table_of_contents_title() }}</option>
    <option value="{{ url_for('table_of_contents', year=year, lang=lang) }}#forward">{{ self.foreword_title() }}</option>
```